### PR TITLE
Issue/2428 comprehensive graph (Connect #2428)

### PR DESCRIPTION
**Pull request by stellanl**
#### Before the PR (what is the issue or what needed to be done)
Comprehensive charts were pie diagrams.
#### The solution
Change them to bar graphs, style them according to Akvo brand book.
## Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: GAE/src/org/akvo/flow/util/JFreechartChartUtil.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+     */
+    public static byte[] getBarChart(List<String> labels, List<String> values,
+            String title, int width, int height) {
+        DefaultCategoryDataset dataset = new DefaultCategoryDataset();
+
+        for (int i = 0; i < labels.size(); i++) {
+            dataset.setValue(
+                    Double.parseDouble(values.get(i)),
+                    "", // All on one nameless row
+                    labels.get(i) //Integer.valueOf(i) //Column
```
Comment (valllllll2000) [inline GAE/src/org/akvo/flow/util/JFreechartChartUtil.java]: What is //Integer.valueOf(i) if it is some old code maybe you can delete it.

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 2]
```diff
@@ -264,9 +264,10 @@ public int compare(String o1, String o2) {
         }
     }
 
-    private Workbook createWorkbookAndFormats(){
+    private Workbook createWorkbookAndFormats() {
+        // This window may be too small for some OPTION questions
+        // on a comprehensive stats sheet
         Workbook wb = new SXSSFWorkbook(100);
```
Comment (valllllll2000) [inline GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: Maybe extracting 100 to a constant with a meaningful name would help understanding what it stands for

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 3]
```diff
@@ -1471,55 +1473,64 @@ private void writeSummaryReport(
...
+                    boolean doChart = false;
+                    boolean doDataTable = false;
+                
+                    if (QuestionType.OPTION == question.getType()) { 
+                        doChart = true;
+                        doDataTable = true;                        
+                    } else if (QuestionType.NUMBER == question.getType()) {
+                        doDescriptiveStats = true;
+                        //skip table and chart (phone numbers etc get ridiculous)
+                    } else { //TODO: add cascade
```
Comment (valllllll2000) [inline GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: Is that part of a future issue? Will the cascades be missing?

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java [hunk 4]
```diff
@@ -1846,8 +1873,8 @@ public static void main(String[] args) {
         Map<String, String> criteria = new HashMap<String, String>();
         Map<String, String> options = new HashMap<String, String>();
 //        options.put(TYPE_OPT, DATA_CLEANING_TYPE);
-        options.put(TYPE_OPT, DATA_ANALYSIS_TYPE);
-//        options.put(TYPE_OPT, COMPREHENSIVE_TYPE);
+//        options.put(TYPE_OPT, DATA_ANALYSIS_TYPE);
```
Comment (valllllll2000) [inline GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: why is this line commented out? If that code is not used maybe it should be deleted?

Comment (stellanl) [reply GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: This whole main() method is only for testing. So you can test the 3 different type by uncommenting one of them.

File: GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java (hunk 3 above)
Comment (stellanl) [reply GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java]: Another issue, yes.

**Pull request merged**